modules/nf-core/bbmap/filterbyname/main.nf

```diff
@@ -0,0 +1,59 @@
+process BBMAP_FILTERBYNAME {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::bbmap=39.01" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/bbmap:39.01--h311275f_0':
+        'quay.io/biocontainers/bbmap:39.01--h311275f_0' }"
+
+    input:
+    tuple val(meta), path(reads)
+    val names
```

Comment:
It is also advised to specify the names like you did with `output_extension`

Reply:
Addressed in the [next commit](https://github.com/nf-core/modules/pull/2206/commits/247b6459f070daf2f02f36a4219d2406c43afa6c)